controllers/homework.js

```diff
@@ -498,17 +539,22 @@ router.get('/new', (req, res, next) => {
 		if (req.query.topic) {
 			assignment.lessonId = req.query.topic;
 		}
+		const schoolId = res.locals.currentSchool;
+		const parentType = 'tasks';
+		const data = await fileStorageInit(schoolId, undefined, parentType, req);
```

Comment:
I await here undefined

Reply:
The check is `if (parentId)` in line 38, isn't it? Do you mean that? That would lead to the same result if you use `null`. What I meant is explained here for example: https://blog.greenroots.info/javascript-undefined-and-null-lets-talk-about-it-one-last-time

From this text: "undefined value is typically set by the JavaScript engine when a variable is declared but not assigned with any values.
null value is typically set by programmers when they want to assign an empty/blank value."